docs/posts/duckdb-for-rag/.gitignore

```diff
@@ -0,0 +1,2 @@
+ibis
```

Comment:
What is being ignore by this?

Reply:
can remove, originally wasn't `shutil.rmtree`ing the Ibis repo after it was cloned, so `gitignore`d it. added the deletion because when previewing the docs, it was building all the docs in the cloned repo too